src/BaGet.Decompiler.Nupkg/Objects/AnalysisNugetAssemblyConverter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using AutoMapper;
+using BaGet.Decompiler.Objects;
+
+namespace BaGet.Decompiler.Nupkg.Objects
+{
+    internal static class AnalysisNugetAssemblyConverter
+    {
+        private static readonly IMapper Mapper;
+
+        static AnalysisNugetAssemblyConverter()
+        {
+            Mapper = new MapperConfiguration(config =>
+            {
+                config.CreateMissingTypeMaps = false;
+                config.CreateMap<AnalysisAssembly, AnalysisNugetAssembly>();
+            }).CreateMapper();
+        }
+
+        public static AnalysisNugetAssembly Convert(AnalysisAssembly source)
+        {
+            return Mapper.Map<AnalysisNugetAssembly>(source);
```

Comment:
What's the value in using `AutoMapper` here? I'm wondering if we can cut the dependency

Reply:
I wanted to keep a seperation between the assembly decompilation, and the Nupkg decompilation. But then I also wanted to add extra info to the decompiled assemblies, such as the framework they targeted - hence this inheriting member.

I suppose we could join the Nupkg feature directly into the decompilation, so that it's a core part of it. It would, however, mean that you could decompile an assembly, by itself, but not know which framework it targeted.

Reply:
I've merged them.